cratedb_django/fields/json.py

```diff
@@ -1,7 +1,58 @@
+from enum import StrEnum
+from enum import auto
+from typing import Literal
+from typing import Optional
+
+
+from cratedb_django.fields import CrateDBBaseField
 from cratedb_django.fields import JSONField
 
 
+class ObjectPolicy(StrEnum):
+    strict = auto()
+    dynamic = auto()
+    ignored = auto()
+
+
 class ObjectField(JSONField):
+    crate_type = "OBJECT"
+
+    def __init__(
+        self,
+        policy: Literal["strict", "dynamic", "ignored"] = "dynamic",
+        schema: Optional[dict[str, CrateDBBaseField]] = None,
+        *args,
+        **kwargs,
+    ):
+        self.policy = ObjectPolicy(policy)
+        self.schema = schema
+        super().__init__(*args, **kwargs)
+
+    def db_type(self, connection):
+        if self.policy == ObjectPolicy.dynamic:
+            return self.crate_type
```

Comment:
Is explicit better than implicit?
```suggestion
        if self.policy == ObjectPolicy.dynamic:
            return f"{self.crate_type}({ObjectPolicy.dynamic})"
```